### PR TITLE
Fix typos in config file

### PR DESCRIPTION
**Pull request by git-hulk**
I find there are many typos in the config file and
fix them.

### CODE REVIEW

File: kvrocks.conf [hunk 1]
```diff
@@ -97,7 +98,7 @@ dir /tmp/kvrocks
 #
 # log-dir stdout
 
-# When running daemonized, kvrocks writes a pid file in ${CONFIG_DIR}/kvrocks.pid by
+# When running in daemonize mode, kvrocks write a PID file in ${CONFIG_DIR}/kvrocks.pid by
```
Comment (tisonkun) [inline kvrocks.conf]:
```suggestion
# When running in daemonize mode, kvrocks writes a PID file in ${CONFIG_DIR}/kvrocks.pid by
```

kvrocks is a singular although it ends with `s`, lol.

File: kvrocks.conf [hunk 2]
```diff
@@ -179,11 +180,11 @@ use-rsid-psync no
 # is still in progress, the slave can act in two different ways:
 #
 # 1) if slave-serve-stale-data is set to 'yes' (the default) the slave will
-#    still reply to client requests, possibly with out of date data, or the
+#    still reply to client requests, possibly with out-of-date data, or the
 #    data set may just be empty if this is the first synchronization.
 #
 # 2) if slave-serve-stale-data is set to 'no' the slave will reply with
-#    an error "SYNC with master in progress" to all the kind of commands
+#    an error "SYNC with master in progress" to all the kinds of commands
```
Comment (tisonkun) [inline kvrocks.conf]:
```suggestion
#    an error "SYNC with master in progress" to all kinds of commands
```

File: kvrocks.conf [hunk 3]
```diff
@@ -367,23 +368,23 @@ auto-resize-block-and-sst yes
 ################################ MIGRATE #####################################
 # If the network bandwidth is completely consumed by the migration task,
 # it will affect the availability of kvrocks. To avoid this situation,
-# migrate-speed is adpoted to limit the migrating speed.
-# Migrating speed is limited by controling the duraiton between sending data,
-# the duation is calculated by: 1000000 * migrate-pipeline-size / migrate-speed (us).
+# migrate-speed is adapted to limit the migrating speed.
```
Comment (tisonkun) [inline kvrocks.conf]:
```suggestion
# migrate-speed is adopted to limit the migrating speed.
```

The meaning seems to be 'adopted'?

**Review by tisonkun — COMMENTED** [review]
Comments inline.

File: kvrocks.conf (hunk 3 above)
Comment (git-hulk) [reply kvrocks.conf]: Yes, my bad.

**Review by tisonkun — APPROVED** [review]

**Pull request merged**